html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style></style>
+    <title>html-forms</title>
+    <style>
+      textarea {
+        width: 250px;
+        height: 150px;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label>Email</label>
+      <br />
+      <input type="email" name="email" required />
+      <br />
+      <label>Why is Coding Cool?</label>
+      <br />
+      <select name="option">
+        <option value="">Select an Option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+      <p>Favorite Language (choose one)</p>
+      <label><input type="radio" name="interest" value="HTML" />HTML</label>
+      <label><input type="radio" name="interest" value="CSS" />CSS</label>
+      <label
+        ><input
+          type="radio"
+          name="interest"
+          value="JavaScript" />JavaScript</label
+      >
+      <p>Would you like to sign up for more information?</p>
+      <label
+        ><input type="checkbox" name="check" value="Signup" />Sign me up!</label
+      >
+      <br />
+      <label>Comments</label>
```

Comment:
The label needs a `for` that matches the `id` of the `input` you want to associate it with